torch/ao/quantization/fx/_model_report/detector.py

```diff
@@ -938,6 +938,11 @@ class OutlierDetector(DetectorBase):
         reference_percentile (float, optional): The denominator to find the relative scale of the 100th percentile
             Should be between 0 and 1
             Default: 0.975
+        statistical_threshold (float, optional): Fraction of batches to determine outliers per channel should be above this
```

Comment:
Sorry I should've commented this last time but I feel the name can be more descriptive. Right now it sounds like it could be a threshold for anything, not necessarily related to fraction of batches. Any thoughts?

Reply:
changed to `fraction_batches_used_threshold` to be more descriptive